lib/jnpr/junos/utils/sw.py

```diff
@@ -1007,8 +1011,11 @@ def poweroff(self, in_min=0):
 
         .. todo:: need to better handle the exception event.
         """
-        cmd = E('request-power-off', E('in', str(in_min)))
-
+        if self._is_linux:
```

Comment:
Again, this line can be changed from if self._is_linux to dev.facts['_is_linux']. This will not result in an additional RPC being send to the device because fact values are cached.